packages/overlays/src/utils/deep-contains.js

```diff
@@ -34,7 +32,7 @@ export function deepContains(el, targetEl) {
         return true;
       }
     }
-    checkChildren(el);
   }
+  checkChildren(el);
```

Comment:
Difference is very subtle it seems. So basically we stopped traversal for `Element`s that are not `HTMLElement `?

Maybe we can combine the ifs on line 30 and 31?

Reply:
Yes it stopped traversal for the ones that were ShadowRoot, and of course those should be traversed as well :D. If's can now be combined indeed, changed it 👍

Reply:
Yeah, I just figured it out indeed :) Ok, nice